provision/aws/rds/aurora_create.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -e
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/aurora_common.sh
+
+# https://cloud.redhat.com/blog/using-vpc-peering-to-connect-an-openshift-service-on-an-aws-rosa-cluster-to-an-amazon-rds-mysql-database-in-a-different-vpc
+EXISTING_INSTANCE=$(aws rds describe-db-instances \
+  --query 'DBInstances[*].[DBInstanceIdentifier]'  \
+  --filters Name=db-instance-id,Values=${AURORA_INSTANCE} \
+  --output text
+)
+if [ -n "${EXISTING_INSTANCE}" ]; then
+  echo "Aurora instance '${AURORA_INSTANCE}:${AWS_REGION}' already exists"
+  exit 1
+fi
+
+# Create the Aurora VPC
+AURORA_VPC=$(aws ec2 create-vpc \
+  --cidr-block ${AURORA_VPC_CIDR} \
+  --tag-specifications "ResourceType=vpc, Tags=[{Key=AuroraCluster,Value=${AURORA_CLUSTER}}]" \
+  --output json \
+  | jq -r '.Vpc.VpcId'
+)
+
+# Create the Aurora Subnets
+SUBNET_A=$(aws ec2 create-subnet \
+  --availability-zone "${AWS_REGION}a" \
+  --vpc-id ${AURORA_VPC} \
+  --cidr-block 192.168.0.0/19 \
```

Comment:
Looks like the CIDR spec is hardcoded here. If I override the default value of `AURORA_VPC_CIDR` this will fail with
```
An error occurred (InvalidSubnet.Range) when calling the CreateSubnet operation: The CIDR '192.168.0.0/19' is invalid.
```

Same for SUBNET_B below.